integ-test/src/test/java/org/opensearch/sql/sql/DateTimeFunctionIT.java

```diff
@@ -444,6 +444,19 @@ public void testWeek() throws IOException {
     week("2000-01-01", 2, 52);
   }
 
+  @Test
+  public void testWeek_Of_Year() throws IOException {
+    JSONObject result = executeQuery("select week_of_year(date('2008-02-20'))");
+    verifySchema(result, schema("week_of_year(date('2008-02-20'))", null, "integer"));
+    verifyDataRows(result, rows(7));
+
+    week("2008-02-20", 0, 7);
```

Comment:
You can update `week` function to get one more argument - function name. Then, you can remove duplicating code.

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/175/commits/6a0c01326cfdecf491b45a0ee1e857d087822235